test/integration/features/basic.feature

```diff
@@ -45,7 +45,7 @@ Feature: Basic
     """
 
   Scenario: User has a pre-configured set of persitence volumnes
-    When executing "oc get pv --as system:admin -o=name" retrying 5 times with wait period of 3 seconds
+    When executing "oc get pv --as system:admin -o=name" retrying 12 times with wait period of 5 seconds
```

Comment:
How come you changed this? Did you have issues with the test failing again? I wonder whether there is maybe a different issue after all?